pom.xml

```diff
@@ -269,7 +293,6 @@
 
       <!-- resources -->
       <plugin>
-        <groupId>org.apache.maven.plugins</groupId>
```

Comment:
Is this ok?

Reply:
Yes, the org.apache.maven.plugins is the default groupId, and many of the other plugins in here leave it off. Best to be consistent.